packages/loki/spec/generic/sortingIndexing.spec.ts

```diff
@@ -294,6 +292,7 @@ describe("sorting and indexing", () => {
       expect(indexVals[13] === "asdf").toEqual(true);
     });
 
+    /*
```

Comment:
Comment out?

Reply:
Added back core logic for that one unit test;  

Before going live with v1.0 I will probably want to reassess date operators either in default "js" operator package or "loki" abstract operator package (which we will make default when upgrading legacy lokijs databases)